.github/pull_request_template.md

```diff
@@ -31,14 +31,16 @@ Distro packaging links:
   - REQUIRED
 - Ubuntu: https://packages.ubuntu.com/
    - REQUIRED
-- Fedora: https://apps.fedoraproject.org/packages/
+- Fedora: https://src.fedoraproject.org/
```

Comment:
```suggestion
- Fedora: https://src.fedoraproject.org/browse/projects/
```

Reply:
I see the same interface at the root as with the specific `/browse/projects/` url. Do you think the more specific url will be more stable?

Reply:
I didn't realize this right away, but it looks different for me because I'm logged in. The default landing page if you're logged in is a search page for the packages you maintain, not the full package list.

The portion of the user base that will be logged into Fedora Pagure is probably really small. I'm OK with leaving this as-is if you're concerned about the URL stability. Your call.

Reply:
oh interesting. It makes sense to do the non-dynamic one then.